standalone/runtime/tests/test_multisig.rs

```diff
@@ -124,6 +138,88 @@ fn integration_test_multisig_vesting() {
                 .unwrap_or_default(),
             vesting_amount
         );
+        assert_eq!(
+            TokensPallet::accounts(account_of(EVE), Token(INTR)),
+            AccountData {
+                free: vesting_amount,
+                reserved: 0,
+                frozen: vesting_amount,
+            }
+        );
+    });
+}
+
+#[test]
+fn integration_test_transfer_to_unvested_multisig() {
```

Comment:
Can you please modify the multi-sig transfer tests. What we want to test is:
1. multi-sig receives funds as vested and unvested balances, say in case 1, 100% is unvested (i.e., frozen) and in case 2, 100% vest (i.e., free)
2. multi-sig sends funds to another multi-sig account

This is the way that we will likely forward the funds to our investors who have the same vesting as us (48 weeks cliff and then 48 weeks vesting).